### PR TITLE
Crosswords header meta restyle

### PR DESCRIPTION
**Pull request by NathanielBennett**
More restyling to the crossword header styles : Now looks like this: 

Mob:
cw-redesign

Dt:
cw-redesign-deskto

### CODE REVIEW

File: static/src/stylesheets/module/content/_content.scss [hunk 1]
```diff
@@ -555,6 +555,25 @@
         min-height: 0;
         padding-bottom: 0;
     }
+
+    .content__labels--crossword & {
+        @include mq(leftCol) {
+            margin-top: ($gs-baseline * 6) + 7;
```
Comment (desbo) [inline static/src/stylesheets/module/content/_content.scss]: missing unit on the 7

Comment (gtrufitt) [inline static/src/stylesheets/module/content/_content.scss]: Might be worth adding a comment here to explain what the adjustment is

File: static/src/stylesheets/module/content/_content.global.scss [hunk 2]
```diff
@@ -120,11 +120,9 @@
     }
 
     .meta__extras--crossword & {
```
Comment (gtrufitt) [inline static/src/stylesheets/module/content/_content.global.scss]: Does it seem odd that the crossword specific styles are not in the crossword stylesheets? I am not sure what the consistent approach is, modifications within the component or modifications in the feature stylesheets?

Comment (NathanielBennett) [inline static/src/stylesheets/module/content/_content.global.scss]: @gtrufitt  Point. I'll try and shift this over ...

**Comment by gtrufitt** [comment]
:+1: other than couple of comments :)

**Pull request merged**